checks.d/elastic.py

```diff
@@ -304,7 +357,16 @@ def _define_params(self, version, cluster_stats):
 
         stats_metrics.update(additional_metrics)
 
-        return health_url, nodes_url, stats_url, pending_tasks_url, stats_metrics
+        # Version specific stats metrics about the primary shards
+        pshard_stats_metrics = dict(self.PRIMARY_SHARD_METRICS)
+
+        if version >= [1, 6, 0]:
+            additional_metrics = self.PRIMARY_SHARD_METRICS_POST_1_0
```

Comment:
is the minimum supported version for these metrics 1.6.0 (as per your if condition) or 1.0 (as per the constant name)